tests/specs/benchmarks/verification.k

```diff
@@ -132,6 +132,10 @@ module VERIFICATION
     rule X <=Int #ceil32(X)     => true   requires 0 <=Int X                       [simplification]
     rule #ceil32(X) <Int X      => false  requires 0 <=Int X                       [simplification]
     rule #ceil32(X) <=Int pow16 => true   requires 0 <=Int X andBool X <=Int pow16 [simplification]
+    
+    // for debugging ecrecoverloop00-sig1-invalid-spec
+    rule A  <Int B => #Not ( { true #Equals B <=Int A } ) [simplification]
+    rule A <=Int B => #Not ( { true #Equals B  <Int A } ) [simplification]
 
```

Comment:
@jinxinglim you probably need to add these lemmsa to `VERIFICATION-HASKELL` module, not the `VERIFICATION` module, so that only the Haskell backend gets them.

The Java backend doesn't really use matching logic symbols like this, and it's not having a problem with this proof anyway.

Reply:
Okay, I have shifted the rules to `VERIFICATION-HASKELL` module.